statsd.go

```diff
@@ -25,110 +26,159 @@ func init() {
 	}
 }
 
-func Timer() timer {
-	return timer{time.Now()}
+// Options holds key/value pairs to be used when calling the functions
+// with the "...WithOptions" suffix.
+type Options struct {
+	// Rate specifies the sampling rate to use.
+	Rate float64
+	// AlwaysSend specifies whether the packet should be sent
+	// to the server even if the specified sampling rate is too low.
+	AlwaysSend bool
+}
+
+// DefaultOptions returns the default options
+func DefaultOptions() *Options {
```

Comment:
Perhaps this could be an exported var instead, allowing the default options to be overwritten

Reply:
Ah, yeah, I guess that works. I moved it to a function because I was trying to set it as a `const`, but making it settable does make sense.